src/UI/Badge.elm

```diff
@@ -160,17 +204,61 @@ renderElement cfg (Badge { content, tone } { brightness }) =
             , background
                 |> Palette.toElementColor
                 |> Background.color
-            , Element.paddingEach { top = 4, bottom = 4, left = 5, right = 3 }
+            , Element.paddingEach { top = 4, bottom = 4, left = 7, right = 6 }
```

Comment:
as I mentioned on slack, is safer to fix the letter-spacing adjustment inside of Text, otherwise we would have to do a lot of micro-adjustments out there, but let's fix it in another pr.